TACC/Core-CMS
# GH-101: Branding Bar

# Overview

Redesign the branding bar.

# Changes

- Migrate `branding_logos` CSS to `c-branding` & `c-logo`.
- In `s-header` style `.c-logo` not `.navbar-brand`.
- Update and format markup for `header_branding` and `header_logo`.
- Comment on now-unused secrets values.

# Testing

1. Build CSS.
2. Load site (with page wider than 576px).
3. Confirm branding bar matches [CMS-Common Components > Brand-Bar](https://xd.adobe.com/view/b37b291a-7798-4814-90f3-7ac63c48a3e0-e628/specs/).
4. Resize page to narrow than 576px.
5. Confirm gap between columns is reduced.
